tree_sitter.gemspec

```diff
@@ -6,27 +6,30 @@ $LOAD_PATH.unshift(lib) if !$LOAD_PATH.include?(lib)
 require 'tree_sitter/version'
 
 Gem::Specification.new do |spec|
-  spec.required_ruby_version = '>= 2.7'
+  spec.required_ruby_version = '>= 3.0'
 
-  spec.authors       = ['Firas al-Khalil']
-  spec.email         = ['firasalkhalil@gmail.com']
+  spec.authors       = ['Firas al-Khalil', 'Derek Stride']
+  spec.email         = ['firasalkhalil@gmail.com', 'derek@stride.host']
   spec.homepage      = 'https://www.github.com/Faveod/ruby-tree-sitter'
   spec.license       = 'MIT'
   spec.name          = 'ruby_tree_sitter'
   spec.summary       = 'Ruby bindings for Tree-Sitter'
   spec.version       = TreeSitter::VERSION
 
+  spec.metadata = {
+    'allowed_push_host' => 'https://rubygems.org',
+    'homepage_uri' => spec.homepage,
+    'source_code_uri' => spec.homepage,
+    'changelog_uri' => spec.homepage,
+    'documentation_uri' => 'https://faveod.github.io/ruby-tree-sitter/',
```

Comment:
I guess I'd have to create that repo?

Reply:
That will be the url for a github pages site for this project. We can wait until we have that setup before adding it to the gemspec though. I'll remove it from the diff

Reply:
There's no need to remove it. I added the action to publish the docs in this https://github.com/Faveod/ruby-tree-sitter/pull/36.